subsys/bluetooth/audio/cap_initiator.c

```diff
@@ -875,6 +873,20 @@ int bt_cap_initiator_unicast_audio_start(const struct bt_cap_unicast_audio_start
 		return -EINVAL;
 	}
 
+	for (size_t i = 0U; i < param->count; i++) {
+		const struct bt_bap_stream *bap_stream =
+			&param->stream_params[i].stream->bap_stream;
+
+		if (!stream_is_in_state(bap_stream, BT_BAP_EP_STATE_STREAMING)) {
+			all_streaming = false;
+		}
+	}
+
+	if (all_streaming) {
+		LOG_DBG("All streams are already in the streaming state");
+		return -EALREADY;
+	}
+
```

Comment:
Looking at this, it looks like the only change is that we will no longer send the start command if all streams already are streaming. Previously then, would the whole command be rejected if any of the streams was in streaming?

The PR claims "start from anywhere", but it looks like this rather is made to avoid sending redundant start command?

Reply:
> Previously then, would the whole command be rejected if any of the streams was in streaming?

Before this PR, if any ASE was not in the idle or Codec configured state, it would return an error, and it was expected that the application would release all the ASEs and try again. 

> The PR claims "start from anywhere", but it looks like this rather is made to avoid sending redundant start command?

There isn't a start command :) It's a CAP procedure that does several BAP procedures for each ASE